ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -208,11 +214,76 @@ StackLayout {
     ColumnLayout {
         spacing: 0
 
-        TopBar {
+        StatusChatToolBar {
             id: topBar
-            z: 60
-            Layout.alignment: Qt.AlignHCenter | Qt.AlignTop
             Layout.fillWidth: true
+
+            property string chatId: chatsModel.channelView.activeChannel.id
+            property string profileImage: appMain.getProfileImage(chatId) || ""
+
+            chatInfoButton.title: {
+              if (chatsModel.channelView.activeChannel.chatType === Constants.chatTypeOneToOne) {
+                  return Utils.removeStatusEns(chatsModel.userNameOrAlias(chatsModel.channelView.activeChannel.id))
+              }
+              return chatsModel.channelView.activeChannel.name
+            }
+            chatInfoButton.subTitle: {
+                switch (chatsModel.channelView.activeChannel.chatType) {
+                    case Constants.chatTypeOneToOne:
+                        return (profileModel.contacts.isAdded(topBar.chatId) ?
+                            profileModel.contacts.contactRequestReceived(topBar.chatId) ?
+                                qsTr("Contact") :
```

Comment:
I am a little confused. I saw many places uses `qsTrId` and now I see `qsTr`. Which one shall be used across all the project?

Reply:
We have a script that replaces all `qsTr`'s with `qsTrId`'s for i18n. So the general rule is to use `qsTr`, especially when you introduce new messages that haven't been i18n'ed before in that location of the file.

Reply:
Alright, so if I get this right, in QML we always use `qsTr` and then the build system will do the replacement.